tests/magma/core/model/test_decorators.py

```diff
@@ -202,59 +202,57 @@ def run(self):
                     pass
 
     def test_tags(self):
-        """Checks 'tags' decorator"""
-        # Define minimal ProcModel that requires a single 'AbstractResource'
-        @tags('keyword1', 'keyword2')
+        """Checks 'tag' decorator"""
+        # Define minimal ProcModel and tag it
+        @tag('keyword1', 'keyword2')
         class TestModel(AbstractProcessModel):
             def run(self):
                 pass
 
-        self.assertListEqual(TestModel.tag_list, ['keyword1', 'keyword2'])
-
-        @tags('keyword1', ['keyword2', 'keyword3'])
-        class TestModel2(AbstractProcessModel):
-            def run(self):
-                pass
-
-        self.assertListEqual(TestModel2.tag_list, ['keyword1', 'keyword2',
-                                                   'keyword3'])
+        self.assertListEqual(TestModel.tags, ['keyword1', 'keyword2'])
 
     def test_tags_subclassing(self):
         """Checks that tags are additive over sub-classing/inheritance"""
 
-        # Define minimal ProcModel that requires a single 'AbstractResource'
-        @tags('loihi-1')
+        # Define minimal ProcModel and tag it with first tag
+        @tag('loihi-1')
```

Comment:
I guess this is just for testing. But there is no need to use tags for Loihi1/2. That's what we have @requires for.

Reply:
yes, this is just for exemplary purposes.